src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/UsageBasedInteropStubManager.cs

```diff
@@ -5,7 +5,9 @@
 using Internal.TypeSystem;
 using Internal.TypeSystem.Interop;
 
+using ILCompiler.Dataflow;
```

Comment:
I think the new usings are not needed now.

Reply:
Both usings are needed for `DiagnosticUtilities` and `DiagnosticId`.